validator/remote/src/main/java/tech/pegasys/teku/validator/remote/typedef/handlers/SendSignedBlockRequest.java

```diff
@@ -30,8 +30,12 @@
 
 public class SendSignedBlockRequest extends AbstractTypeDefRequest {
 
+  private final ResponseHandler<Object> responseHandler =
+      new ResponseHandler<>()
+          .withHandler(SC_UNSUPPORTED_MEDIA_TYPE, this::handleUnsupportedResponse);
+
   private final boolean preferSszBlockEncoding;
-  private boolean unsupportedMediaType = false;
+  private volatile boolean unsupportedMediaType = false;
```

Comment:
I'm not sure how often this client is likely to be used, but potentially unsupportedMediaType disappears, and preferSszBlockEncoding becomes an atomic boolean